modules/fishing.py

```diff
@@ -30,29 +24,37 @@
 shark = emoji.emojize(':shark:')  # 98 - 100 100d
 
 
-class FishingSession:
-    id: int
-    state: int
+class FishingState(Enum):
+    WAITING_FOR_BITE = 1
+    FISH_BITING = 2
+    HOOKING = 3
 
 
-async def _bite_timer(ctx: Context, fisher: int):
+sessions: Dict[int, FishingState] = {}
+
 
+async def _bite_timer(ctx: Context, fisher: int):
     time_to_bite = random.uniform(MIN_BITE_TIME, MAX_BITE_TIME)
     await asyncio.sleep(time_to_bite)
 
-    if sessions[fisher].state == WAITING_FOR_BITE:
-        sessions[fisher].state = FISH_BITING
+    if sessions[fisher] == FishingState.WAITING_FOR_BITE:
+        sessions[fisher] = FishingState.FISH_BITING
         await ctx.send(f"**BITE!** {ctx.author.mention} has a fish on their line! Send **!hook** to catch.")
         await _hook_timer(ctx, fisher)
+    else:
+        # error??
+        pass
```

Comment:
`pass` here just means "do nothing". Since Python is an indent-specific language and doesn't use curly braces `{}`, we have to use a keyword here to silently close the `else` block.

We may want to add a log statement here to catch if this was somehow called from a bad state (aka `state != `FishingState.WAITING_FOR_BITE`)